create/templates/shared/web_pixel_extension/index.js

```diff
@@ -1,5 +1,5 @@
 shopify.extend(
-  "Analytics",
+  "WebPixel::Render",
   (api) => {
     const { config, subscribe, unsubscribe } = api;
```

Comment:
```suggestion
    const { analytics, context, configuration } = api;
    const { subscribe } = analytics;
```

Optional: A change to the js API should ship soon, so it might be good to change accordingly now. Also the current name for a page view event is 'page_viewed'